docs/source/getting-started/security-basics.rst

```diff
@@ -1,12 +1,12 @@
-Security settings
```

Comment:
Please keep section headings in "Sentence case" not "Title Case" (here and everywhere below)

Reply:
Done. Thank you...